core/src/test/java/com/squarespace/template/plugins/platform/ContentFormattersTest.java

```diff
@@ -229,4 +229,34 @@ public void testWidthHeight() throws CodeException {
     assertFormatter(HEIGHT, json, "200");
   }
 
+  @Test
+  public void testWebsiteColor() throws CodeException {
+    String template = "{valid-hsla|website-color}";
+    String validHslaJson = "{\"valid-hsla\": {\"hue\": 200, \"saturation\": 0.3, \"lightness\": 0.5, \"alpha\": 0.9}}";
```

Comment:
You can do this a bit more concisely with:
```java
private static final Formatter WEBSITE_COLOR = new WebsiteColorFormatter();
...
assertFormatter(WEBSITE_COLOR, validHslaJson, "hsla(200, 30%, 50%, 0.9)");
```

Reply:
thanks, updated!